src/DotNetLightning.Core/Serialize/Msgs/Msgs.fs

```diff
@@ -1477,3 +1484,19 @@ type GossipTimestampFilter = {
             ls.Write(this.FirstTimestamp, false)
             ls.Write(this.TimestampRange, false)
             
+[<CLIMutable>]
+type MonoHopUnidirectionalPayment = {
+    mutable ChannelId: ChannelId
+    mutable Amount: LNMoney
```

Comment:
is mutable really needed here???

Reply:
All the other message types have it, though I don't know why. Removing it gives a pile of compilation errors.

Reply:
i guess the real way to fix this is change `ILightningSerializable<'T>`'s Deserialize method to return `ILightningSerializable<'T>` instead of unit, right?

Reply:
I think that would be sufficient, since I can't see anywhere else in the code where the fields are being mutated. Returning a value rather than mutating an argument would be more "functional" too.

Reply:
ok I've added an issue about this: https://github.com/joemphilips/DotNetLightning/issues/87